securedrop_client/gui/widgets.py

```diff
@@ -1478,7 +1478,7 @@ def update(self) -> None:
         icon.setPixmap(load_image('file.png'))
 
         if self.file.is_downloaded:
-            description = QLabel("Open")
+            description = QLabel(self.file.original_filename or self.file.filename)
```

Comment:
we should `html.escape` this text, since the original filename is user controlled (HTML injection vector)

Reply:
Done.